migrations/educandu-2022-01-21-01-add-render-media-to-markdown-and-annotation.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2022_01_21_01_add_render_media_to_markdown_and_annotation {
```

Comment:
You copied the `initial-schema` script and then transformed the original into this current script and now there's a `educandu-2022-01-17-01-initial-schema copy.js` in place of the original 😬 

I've done this myself a couple of times 😅

Reply:
:scream: I should check the changed files more carefully, fixed